docs/content/geoh5_format/analyst/objects.rst

```diff
@@ -1,21 +1,28 @@
+.. _analyst_objects:
```

Comment:
All these changes are reverting an older commit that fixed rstcheck for us. Can you revert your changes here?